server.go

```diff
@@ -87,7 +88,7 @@ type server struct {
 
 	// nodeSigner is an implementation of the MessageSigner implementation
 	// that's backed by the identity private key of the running lnd node.
-	nodeSigner *nodeSigner
+	nodeSigner *netann.NodeSigner
```

Comment:
nit: ideally should be done in same commit as rename to ensure compilation is intact

Reply:
split into add, rename, remove to maintain compilation and distinct files for rebasability. Final diff should appear the same as a rename

Reply:
yeah seems the only way if we want to try to keep the one commit/file :S